src/api/public/apidocs-new/paths/source_projects.yaml

```
@@ -0,0 +1,44 @@
+get:
+  summary: List all the projects
+  description: |
+    Get a list of projects or a list of deleted projects.
+  parameters:
+    - name: deleted
+      in: query
+      schema:
+        type: integer
+      description: 0 for existing projects and 0 for deleted projects.
+      example: 0
+      required: false
+
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/source/projects.yaml'
+    '400':
+      description: Wrong value for deleted parameter.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            type: object
+            properties:
+              code:
+                type: string
+                xml:
+                  attribute: true
+              summary:
+                type: string
+            xml:
+              name: status
```

```suggestion
            $ref: '../schemas/api_response.yaml'
```
It should be possible to use the `api_response.yaml` schema here.

---

`api_response.yaml` has more information than we need here. We just need code and summary

---

The schema defines all possible fields and how they are arranged. But it doesn't mean that we have to use all the fields, in all the examples.

If in this case the example uses ony `code` and `summary`, there is no need to define another schema inline. When we don't assign values to the other fields, those fields are simply not shown.

---

I see. Okay, I've updated it